react/features/base/dialog/components/native/CustomDialog.js

```diff
@@ -19,4 +21,19 @@ class CustomDialog extends BaseDialog<Props, *> {
     }
 }
 
-export default connect()(CustomDialog);
+/**
+ * Maps part of the Redux state to the props of this component.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {{
+ *     _colorSchemedStyles: StyleType
+ * }}
+ */
+function _mapPropsToState(state) {
+    return {
+        _colorSchemedStyles:
+            ColorSchemeRegistry.get(state, 'features/base/dialog')
```

Comment:
I see we are using this in many places... maybe it's time to have the feature name in a constant